force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -378,6 +495,17 @@ public with sharing class ClassesObjectsMethods {
      * @return The Person object of the older person.
      */
     public static Person getOlderPerson(Person person1, Person person2) {
-        return null; // Replace null with the variable you used to store the result
+    if (person1.age > person2.age) {
+        return person1;
+    }
+    
+    else if (person1.age < person2.age) {
+        return person2;
+    }
+    
+    else {     
+     
+        return person1; // Replace null with the variable you used to store the result
```

Comment:
If you started with the `>=` operators in the first if statement you could have shortened your code dramatically. Do you see how?